projects/xregistry/README.md

```diff
@@ -5,7 +5,7 @@ This directory contains artwork in standard formats for XRegistry. We've prepare
 
 - horizontal (also known as landscape format),
 - stacked (which is closer to square), and
-- icon (which does not include the name and is square).
+- icon (which has the simplified logo over of the project name).
```

Comment:
oops, "of" shouldn't be there

Reply:
I also goofed on the line to be changed. I've corrected both now 😅